build-aux/snap/snapcraft.yaml

```diff
@@ -420,14 +445,37 @@ parts:
       # snap should include the information about a particular openssl version
       # which was included in the build but the core22 FIPS variant snap has
       # been stripped off of the manifest.
-      if [ -f fips-build ]; then
+      if [ -f "${CRAFT_PROJECT_DIR}/fips-build-lp" ]; then
+        mkdir fips-libs-from-debs
+        mkdir fips-libs-from-debs-stage
+
+        # extract
+        for f in fips-debs/*.deb; do
+          dpkg -x "$f" fips-libs-from-debs
+        done
+
+        # and cherry pick the files we want
+        (
+          cd fips-libs-from-debs
+
+          # XXX note with -print0 only the last match is printed
```

Comment:
Given that the code below is not using `-print0` this is somewhat confusing.

In addition, this is not true, at least not on my system.

Reply:
well it didn't work on 22.04

Reply:
turns out I'm a muppet and I can't use `find` properly